stripe-test-e2e/src/test/java/com/stripe/android/test/e2e/Request.kt

```diff
@@ -1,25 +1,30 @@
 package com.stripe.android.test.e2e
 
 import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
 import com.stripe.android.core.ApiVersion
 
 sealed class Request {
+    @JsonClass(generateAdapter = true)
```

Comment:
Behavior change, new versions require kotlin reflect, or code gen. I chose the code gen route.